frontend/public/components/storage/create-pvc.tsx

```diff
@@ -155,29 +155,22 @@ export const CreatePVCForm: React.FC<CreatePVCFormProps> = (props) => {
   };
 
   const handleStorageClass = (updatedStorageClass) => {
+    const provisioner: string = updatedStorageClass?.provisioner || '';
```

Comment:
Does optional chaining work in Firefox or IE?

Reply:
It's suppored in TypeScript. The final JS won't have it.